src/test/java/core/transactions/OperationHandlerTest.java

```diff
@@ -0,0 +1,61 @@
+package core.transactions;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.exception.OperationHandlerException;
+import org.junit.jupiter.api.Test;
+
+public class OperationHandlerTest {
+    private static final OperationHandler handler = new BalanceOperationHandler();
+
+    @Test
+    public void testBalanceOperationHandler_ok() {
+        int currentAmount = 100;
+        int operationAmount = 50;
+        int result = handler.getTransaction(currentAmount, operationAmount);
+        assertEquals(150, result);
```

Comment:
```suggestion
        assertEquals(currectAmount + operationAmoung, result);
```
or expectedAmount = currectAmount + operationAmoung; then assertEquals(expectedAmount , result);